arrow-core/src/main/kotlin/arrow/core/extensions/const.kt

```diff
@@ -107,7 +107,7 @@ interface ConstTraverse<X> : Traverse<ConstPartialOf<X>>, ConstFoldable<X> {
   override fun <T, U> ConstOf<X, T>.map(f: (T) -> U): Const<X, U> = fix().retag()
 
   override fun <G, A, B> ConstOf<X, A>.traverse(AP: Applicative<G>, f: (A) -> Kind<G, B>): Kind<G, ConstOf<X, B>> =
-    fix().traverse(AP, f)
+    fix().traverse(AP)
```

Comment:
Not removing the param from the declaration also here? should we deprecate it at least?